mlflow/metrics/utils/prompts/v1.py

```diff
@@ -75,3 +75,30 @@ def to_dict(self):
 
     def _format_examples(self):
         return "\n".join(map(str, self.examples))
+
+
+@dataclass
+class CorrectnessMetric:
+    definition = (
+        "Correctness refers to how well the generated output matches "
+        "or aligns with the reference or ground truth text that is considered "
+        "accurate and appropriate for the given input. The ground truth serves as "
+        "a benchmark against which the provided output is compared to determine the "
+        "level of accuracy and fidelity."
+    )
+
```

Comment:
should we be providing an example by default? my understanding is a high quality judge needs the example, but in this current setup it is optional. cc @sunishsheth2009 @suanmiao

Reply:
Yes we can provide one example by default. but users can override the example list or pass in None if they really don't want to pass in an example to LLM. 
Also that would serve as an example to the users on what the expectations for an example are.